pkg/services/accesscontrol/migrator/migrator.go

```diff
@@ -0,0 +1,44 @@
+package migrator
+
+import (
+	"context"
+	"time"
+
+	"github.com/grafana/grafana/pkg/infra/db"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+)
+
+func MigrateScopeSplit(db db.DB, log log.Logger) error {
+	t := time.Now()
+	var count = 0
+	err := db.WithTransactionalDbSession(context.Background(), func(sess *sqlstore.DBSession) error {
+		var permissions []accesscontrol.Permission
+
+		err := sess.SQL("SELECT * FROM permission WHERE NOT scope = '' AND kind = '' AND attribute = '' AND identifier = ''").Find(&permissions)
```

Comment:
Given how split scope is written, we'll always have an identifier if the scope is not empty, so  I think there is no need to check kind and attribute.
```suggestion
		err := sess.SQL("SELECT * FROM permission WHERE NOT scope = '' AND identifier = ''").Find(&permissions)
```

Reply:
Yeah, I was thinking about that too, but this felt safer in case we change scopes in the future. But I guess that is unlikely 🤔